vision/obstacle/obstacle_tracker.py

```diff
@@ -94,7 +94,7 @@ def update(self, new_obstacle_boxes: list) -> None:
         new_obstacles = np.array([])
 
         # transform the list of bounding boxes into an array of Obstacles
-        for i in np.arange(len(new_obstacle_boxes)):
+        for i in np.arange(np.size(new_obstacle_boxes)):
```

Comment:
This should be len instead of size. size propagates down the dimensions of the array so it will get a different value than len. Changed to len by #247

Reply:
I'm not actually sure when or how this got changed but it was unintentional. I noticed it in the merge conflict and didn't think much of it. Thanks.